src/main/webapp/css/srp_style.css

```diff
@@ -37,33 +37,68 @@ h1 {
 
 .card {
   margin-bottom: 20px;
-  padding: 24px;
+  padding: 24px 16px 24px 24px;
```

Comment:
Why? This looks sketchy.

Reply:
The scroll bar on the right takes up 8 px of the container? But a pretty minor UI thing I guess.

Reply:
I should point out that this is particular to Chrome. (See the non-standard "overflow: overlay" property to see how you could make Chrome and other Chrome-like browsers not make the scrollbar take up additional space, and also "scrollbar-gutter" for the \~future~ of controlling this.) Other browsers may not have this - for example, Firefox on Mac puts the scrollbar on top of the padding, so it looks off-center there.